maas/client/viscera/machines.py

```diff
@@ -171,6 +172,10 @@ class MachineType(ObjectType):
         data = await cls._handler.read(system_id=system_id)
         return cls(data)
 
+    async def details(cls, system_id):
+        data = await cls._handler.details(system_id=system_id)
+        return bson.loads(data)
```

Comment:
This is better for my usecase actually, as I get the list of machines and iterate on them I can do:

machines = client.machines.list()
for node in machines:
    node.details()

Thanks!